tests/layer_tests/onnx_tests/test_prelu.py

```diff
@@ -141,8 +141,6 @@ def test_prelu_opset7_shared_channels(self, params, ie_device, precision, ir_ver
     @pytest.mark.precommit
     def test_prelu_shared_channels_precommit(self, params, ie_device, precision, ir_version,
                                              temp_dir, use_old_api):
-        if ie_device == 'GPU':
-            pytest.xfail("To be checked in scope of CVS-95606")
```

Comment:
no more other changes are needed to correct slope shape?

Reply:
Test changes will be in another PR

Reply:
Got it